.infrastructure/configMap.yml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: todoapp-config
+  namespace: todoapp
+date:
+  PYTHONUNBUFFERED: "1"
```

Comment:
Add Eof(End-of-file), need to add the empty line at the end
![image](https://github.com/mate-academy/devops_todolist_kubernetes_task_7_configuration_management/assets/87720583/6e1f447e-8614-421f-a4ab-65e570f9c48f)